run_dir/design/worksets.html

```diff
@@ -129,7 +128,7 @@ <h2 class="closed_ws_page_heading"><small class="text-muted">All worksets with p
   </div>
 </div>
 <!-- #page_content -->
-<script src="/static/js/jquery.dataTables.min.js"></script>
+<script src="/static/js/dataTables-extensions-1.13.6.min.js"></script>
 <script src="/static/js/worksets.js?v={{ gs_globals['git_commit'] }}" id="worksets-js" data-worksets="{{worksets}}"></script>
 <script src="/static/js/clipboard.min.js"></script>
```

Comment:
The buttons been implemented here?

Reply:
The script has been added, but not the code

Reply:
I can't test it locally because the data isn't there as we know, but I could add the code?

Reply:
What I meant was, this can be removed here? `<script src="/static/js/clipboard.min.js"></script>`

You've already added the buttons for copy and excel in worksets.js

I'm not sure what data we need to test it? It gets data from statusDB.

Reply:
If you meant the LIMS queue, that one uses `workset_queues`

Reply:
I meant for the qPCR, the worksets page gets the data locally 😆 

Ah, the worksets also has the clipboard script, right, thought you meant for qPCR